README.md

```diff
@@ -13,51 +28,36 @@ Getting started
 1. If you are using the `develop` branch of vector, you will probably need to
    rebuild one of the dependencies, due to https://github.com/npm/npm/issues/3055:
    `(cd node_modules/matrix-react-sdk && npm install)`
-1. Start the development builder and a testing server: `npm start`
-1. Wait a few seconds for the initial build to finish (the command won't
-   terminate: it's running a web server for you).
-1. Open http://127.0.0.1:8080/ in your browser to see your newly built Vector.
+1. Configure the app by copying `config.sample.json` to `config.json` and modifying
```

Comment:
True - we could put it in the root directory, although then it's unclear that it needs to be copied to the `vector/` directory. Another option would be to exclude `config.sample.json` from the tarball explicitly.